src/pages/auth/login.tsx

```diff
@@ -29,12 +30,25 @@ const LoginPage = () => {
 
     try {
       const response = await UserService.login({ email, password });
-      if (response.message === 'Incorrect email or password') {
+      emailValidation(email, setEmailInvalid);
+      // create e toast message;
+      passwordValidation(password, setPasswordInvalid);
+      // create a toast message that say Password must have min 5 and max 20 characters with 1 letter, 1 number and 1 special character;
+      if (response.message) {
```

Comment:
If the backend start to return a message when success, this code will work?
We should check by the object we want to use: `response.user`.

Reply:
probably its a better solution. Thanks